gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,12 +1,6 @@
-#
-#  Copyright OpenSearch Contributors
-#  SPDX-License-Identifier: Apache-2.0
-#
-
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.1-bin.zip
+networkTimeout=10000
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionSha256Sum=97a52d145762adc241bad7fd18289bf7f6801e08ece6badf80402fe2b9f250b1
```

Comment:
can we have sha256 for new version of gradle?

Reply:
Good point. Updated.